Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -10539,7 +10539,15 @@ bool WebPageProxy::useGPUProcessForDOMRenderingEnabled() const
 
 WebPageCreationParameters WebPageProxy::creationParameters(WebProcessProxy& process, DrawingAreaProxy& drawingArea, WebCore::FrameIdentifier mainFrameIdentifier, std::optional<RemotePageParameters>&& remotePageParameters, bool isProcessSwap, RefPtr<API::WebsitePolicies>&& websitePolicies)
 {
-    WebPageCreationParameters parameters;
+    std::reference_wrapper<WebUserContentControllerProxy> userContentController(m_userContentController.get());
```

Comment:
This seems a bit odd, why not using a Ref? creationParameters is probably not called that often anyway.

Reply:
I merely moved the code but I'll look into making this change.